tartufo/scanner.py

```diff
@@ -343,11 +342,61 @@ def find_strings(
             branch_name,
         )
         output = handle_results(output, output_dir, found_issues)
-    output["project_path"] = project_path
-    output["clone_uri"] = git_url
+    output["project_path"] = repo_path
```

Comment:
I'm generally a fan of this cleanup, but do we assume that the caller will be able to identify/guess the remote repository URL from the generated scratch path? That is, are we losing useful information by no longer stashing git_url in the findings?

Reply:
That is a good call out. My main check was to see if these were used anywhere else in this code base, and they are not. I didn't think about this being included in the actual json files which get written out, and whether it would be useful there. @mxhenry do you know if these values are currently used by any of our tooling?

I think this would be good to include in our refactoring of issues/output to a class-based structure spawned via #23.

Reply:
Replied in email

Reply:
Glancing through our current usage, it doesn't look like we're using those pieces right now. I think it is useful to have though, so we should worry about including it in the 2.0 refactoring.